pkg/service/record/record.go

```diff
@@ -29,28 +29,30 @@ func NewRecorder(logger *zap.Logger) Recorder {
 	}
 }
 
-func (r *recorder) CaptureTraffic(path string, proxyPort uint32, appCmd, appContainer, appNetwork string, Delay uint64, buildDelay time.Duration, ports []uint, filters *models.TestFilter, enableTele bool, passThroughHosts []models.Filters) {
-
+func (r *recorder) CaptureTraffic(params TrafficCaptureParams) {
 	var ps *proxy.ProxySet
+
 	stopper := make(chan os.Signal, 1)
```

Comment:
params isn't a suitable name, any better name can be used

Reply:
Refactored to CaptureTrafficConfig